modules/apps/object/object-service/src/main/java/com/liferay/object/internal/uad/display/ObjectEntryUADDisplay.java

```diff
@@ -0,0 +1,196 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.internal.uad.display;
+
+import com.liferay.object.internal.uad.constants.ObjectUADConstants;
+import com.liferay.object.model.ObjectDefinition;
+import com.liferay.object.model.ObjectEntry;
+import com.liferay.object.scope.ObjectScopeProvider;
+import com.liferay.object.scope.ObjectScopeProviderRegistry;
+import com.liferay.object.service.ObjectEntryLocalService;
+import com.liferay.petra.string.CharPool;
+import com.liferay.portal.kernel.dao.orm.DynamicQuery;
+import com.liferay.portal.kernel.dao.orm.OrderFactoryUtil;
+import com.liferay.portal.kernel.dao.orm.RestrictionsFactoryUtil;
+import com.liferay.portal.kernel.portlet.LiferayPortletRequest;
+import com.liferay.portal.kernel.portlet.LiferayPortletResponse;
+import com.liferay.portal.kernel.portlet.url.builder.PortletURLBuilder;
+import com.liferay.portal.kernel.service.GroupLocalService;
+import com.liferay.portal.kernel.util.ArrayUtil;
+import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.kernel.util.Portal;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portal.kernel.util.Validator;
+import com.liferay.user.associated.data.display.BaseModelUADDisplay;
+
+import java.io.Serializable;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+
+import javax.portlet.PortletRequest;
+
+/**
+ * @author Carolina Barbosa
+ */
+public class ObjectEntryUADDisplay extends BaseModelUADDisplay<ObjectEntry> {
+
+	public ObjectEntryUADDisplay(
+		GroupLocalService groupLocalService, ObjectDefinition objectDefinition,
+		ObjectEntryLocalService objectEntryLocalService,
+		ObjectScopeProviderRegistry objectScopeProviderRegistry,
+		Portal portal) {
+
+		_groupLocalService = groupLocalService;
+		_objectDefinition = objectDefinition;
+		_objectEntryLocalService = objectEntryLocalService;
+		_objectScopeProviderRegistry = objectScopeProviderRegistry;
+		_portal = portal;
+	}
+
+	@Override
+	public ObjectEntry get(Serializable primaryKey) throws Exception {
+		return _objectEntryLocalService.getObjectEntry(
+			Long.valueOf(primaryKey.toString()));
+	}
+
+	@Override
+	public String[] getDisplayFieldNames() {
+		return new String[] {"externalReferenceCode", "objectEntryId"};
```

Comment:
No.. there were so many details that I needed to implement and that were not defined in the Acceptance Criteria that I thought would be easier if the QAs and Product team could validate the story in this PR, since we are going to send it without FF. What do you think?